ui/app/templates/jobs/run/templates/template.hbs

```diff
@@ -0,0 +1,76 @@
+{{page-title "Edit template"}}
+<section class="section">
+    <header class="run-job-header">
+        <div>
+            <h1 class="title is-3">Edit template</h1>
+        </div>
+        <Hds::Dropdown as |dd|>
+            <dd.ToggleButton @text="Manage" data-test-manage-dd />
+            <dd.Interactive @icon="trash" @text="Delete" @color="critical" {{on "click" this.toggleModal}} data-test-delete />
+        </Hds::Dropdown>
```

Comment:
If we're never planning on having a second button here, could we save the operator a click and put the Delete button outside of it?

Reply:
I'm matching the pattern that we see in `cloud-ui`, `vault` and `boundary` here.

I'm imagining we will eventually be adding to this manage dropdown because we may decide that we want to be a read-only view by default across the application now that we have more editing scenarios or say we want to allow additional key value pairs to be added later on.

Generally, I like the pattern of the managerial buttons being grouped and stored together in a nice dropdown.

<img width="1647" alt="Screen Shot 2023-01-20 at 11 34 56 AM" src="https://user-images.githubusercontent.com/41024828/213753342-c28b79a6-205c-4219-8aad-8a950c1822aa.png">

<img width="1110" alt="Screen Shot 2023-01-20 at 11 34 30 AM" src="https://user-images.githubusercontent.com/41024828/213753315-e228537c-1c01-4868-b211-4da83cf4a71a.png">

Reply:
Right, but the button in boundary, for example, has further examples other than just Delete (has Add Members etc.)

I like dropdown buttons for sets of actions too, but it's adding unnecessary overhead if there's just one thing you can do beneath it.

Reply:
Also just noting: is there no way in HDS for me to use keyboard navigation in a dropdown? Opening the Manage button here doesn't give me the ability to select Delete with my keyboard that I can tell.

Reply:
I'm successfully using the normal pattern for using tabs to access the manage dropdown. But this sounds like a question to drop into the `#design-systems` Slack channel. I'm typing `tab` + `enter` + `tab` + `enter` to complete the entire action of deleting a template.

Reply:
> Right, but the button in boundary, for example, has further examples other than just Delete (has Add Members etc.)
>
> I like dropdown buttons for sets of actions too, but it's adding unnecessary overhead if there's just one thing you can do beneath it.

OK, no worries! 😅 I don't have any strong opinions here -- to clarify:  are you saying that you want me to change this because you don't like it? I'm totally happy to accommodate here! 

If you don't like this pattern I can just put a delete button next to the edit button. Is that what you're looking for?

Its a little challenging for me to move forward on this PR because I'm unsure of about the language of the comment. It would be a little more helpful for me to know if we're having a discussion or if this is something that you're requesting a change on.

Just tell me what you're looking for and I'll gladly make the change! 😄

Reply:
> If you don't like this pattern I can just put a delete button next to the edit button. Is that what you're looking for?
Am I missing something? I don't see any edit button for this: 
<img width="1200" alt="image" src="https://user-images.githubusercontent.com/713991/213899622-3f254df0-e0ee-4b29-8558-cddc44ec1211.png">

I'd like to just see the delete button here, no manage button that you have to click first.